src/main/java/stirling/software/SPDF/config/AppUpdateService.java

```diff
@@ -0,0 +1,19 @@
+package stirling.software.SPDF.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+@Service
+class AppUpdateService {
+
+    @Autowired(required = false)
+    ShowAdminInterface showAdmin;
+
+    @Bean(name = "shouldShow")
+    @Scope("request")
+    public boolean shouldShow() {
+        return (showAdmin != null) ? showAdmin.getShowUpdateOnlyAdmins() : true;
```

Comment:
shouldnt there be boolean showUpdate = applicationProperties.getSystem().getShowUpdate(); here so people not using auth can still use this setting?

Reply:
Exactly, people who work without logging in should also have the option to hide or show the update message

Reply:
So you need to split the applicationProperties.getSystem().getShowUpdate();
and the admin one between AppUpdateService.java and the ShowAdmin
Like i showed
![image](https://github.com/Stirling-Tools/Stirling-PDF/assets/77850077/b115afbf-3437-4332-9792-40a110c30051)

Reply:
Do you have any concerns? I tested it with admin and normal user as well as with the "no login" option.

Reply:
when running without security jar itself this will never read the showUpdate in its current form,
showAdmin would be null and hence always default to true